src/app.scss

```diff
@@ -34,81 +38,97 @@ body {
 }
 
 :root {
-	/* TODO properly name these */
 	--main-font: 'Manrope', sans-serif;
 	--mono-font: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace;
-	--white: hsl(206, 100%, 94%);
-	--accent-color: hsl(206, 100%, 81%);
-	--accent-color-two: hsl(208, 75%, 82%);
-	--accent-low-opacity: hsla(205, 91%, 69%, 0.15);
-	--bg-color: hsl(252, 10%, 11%);
-	--grey-one: hsl(210, 14%, 17%);
-	--grey-two: hsl(212, 19%, 19%);
-	--grey-three: hsl(221, 17%, 26%);
-	--grey-four: hsl(226, 48%, 18%);
-	--grey-five: hsl(208, 30%, 75%);
-	--grey-six: hsl(230, 9%, 13%);
-	--grey-seven: hsl(240, 9%, 13.5%);
-	--grey-eight: hsla(207, 30%, 75%, 0.577);
-	--grey-nine: hsla(240, 6%, 7%, 0.3);
-	--grey-ten: hsl(230, 9.5%, 17.5%);
-	--grey-eleven: hsl(208, 10%, 40%);
+
+	--text-one: hsl(calc(var(--hue, 206) + 0), 100%, 94%);
+	--surface-one: hsl(calc(var(--hue, 206) + 0), 100%, 94%);
+
```

Comment:
Remove the unnecessary empty lines from here, move and order the same groups of colors together.

Reply:
the new lines exist to separate variables with the same values, making them easier to maintain